src/cart/README.env

```diff
@@ -80,6 +80,11 @@ This file lists the environment variables used in CaRT.
    by default, and can be specified using DD_SUBSYS, for example:
    "DD_SUBSYS=RPC,BULK,CORPC,GRP,LM,HG,PMIX,ST,IV" or also "DD_SUBSYS=all".
 
+ . D_MEMORY_TRACK
+   User can enable memory track by D_MEMORY_TRACK=1. With the environment, all
+   of allocations inside DAOS (by D_ALLOC) will be tracked, and total allocated
+   bytes per xstream can be shown through metrics.
+
```

Comment:
This is server-only from looking at the code?  The implementation is certainly only in server code and I think metrics are server-only as well currently?

Reply:
yes, server only, since metrics are currently only supported on the server side.